content/blog/data-processing-compliance-statements-and-sla.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Data Processing, Compliance Statements and SLA
+author: Arnout Engelen
+date: 2024-07-10
+---
+
+The Apache Software Foundation is a non commercial, open source organization that relies on a large community of volunteers and companies to maintain its software. It provides this software free (and gratis) 'as is' - as per [its license](https://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
Is "free" not the same as "gratis" :).

BTW. Maybe a good context to mention "free as a puppie" here. You get it for free, you can do what you want with it, but it's also your responsibility to take care about it :).